pyproject.toml

```diff
@@ -376,7 +376,7 @@ show_deps = true
 [tool.codespell]
 # local codespell matches `./docs`, pre-commit codespell matches `docs`
 skip = "*.lock,.direnv,.git,./docs/_freeze,docs/_freeze/**,*.svg,*.css,*.html,*.js"
-ignore-regex = '\b(i[if]f|I[IF]F|AFE)\b'
+ignore-regex = '\b(i[if]f|I[IF]F|AFE|inout)\b'
```

Comment:
Would be great if inline ignores were supported, but [they're not](https://github.com/codespell-project/codespell/issues/1212).